docs/2022/puzzles/day07.md

```diff
@@ -1,14 +1,149 @@
 import Solver from "../../../../../website/src/components/Solver.js"
 
 # Day 7: No Space Left On Device
+code by [Jan Boerman](https://twitter.com/JanBoerman95)
 
 ## Puzzle description
 
 https://adventofcode.com/2022/day/7
 
+## Solution
+
+First of all, we need to come up with a way to parse out input code:
```

Comment:
Id swap the definition + explanation of the Command enum and the input parsing, I think it works better if you don't have forward references